src/tracing/errors.ts

```diff
@@ -69,9 +69,9 @@ function shouldRejectError(error: EventHint['originalException']) {
     // Content security policy 'unsafe-eval' errors can be filtered out because there are expected failures.
     // For example, if a user runs an eval statement in console this error would still get thrown.
     // TODO(WEB-2348): We should extend this to filter out any type of CSP error.
-    if (error.message.match(/'unsafe-eval'.*content security policy/i)) {
-      return true
-    }
+    if (error.message.match(/'unsafe-eval'.*content security policy/i)) return true
+    if (error.message.match(/Blocked a frame with origin ".*" from accessing a cross-origin frame./)) return true
+    if (error.message.match(/NotAllowedError: Write permission denied./)) return true
```

Comment:
Can we create a global array of error messages to filter out and then just check if current message is present?

Reply:
Talked in DM but think we should handle that in a separate PR, I can make a task for it.